app/src/main/res/layout/activity_event_review.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
Seems like this file is not used, is it normal ?

Reply:
Yes! I was preparing the fragment but I decided to do a first PR to split the work. In my next PR I will be using it.